packages/graph/hash_graph/tests/rest-test.http

```diff
@@ -319,7 +319,7 @@ Accept: application/json
 %}
 
 ### Get Person entity
-GET http://127.0.0.1:4000/entities/{{person_a_entity_id}}
+GET http://127.0.0.1:4000/entities/fdsfds
```

Comment:
This was a test that was not supposed to be pushed. It gots committed while rebasing.
```suggestion
GET http://127.0.0.1:4000/entities/{{person_a_entity_id}}
```